cmd/microcloud/preseed.go

```diff
@@ -37,6 +42,8 @@ type Preseed struct {
 // System represents the structure of the systems we expect to find in the preseed yaml.
 type System struct {
 	Name            string      `yaml:"name"`
+	ListenAddress   string      `yaml:"listen_address"`
```

Comment:
This isn't strictly the "listen_address" right? it's also the actual cluster address that will be used. Maybe just call it `address`?

Reply:
Fixed.